doc/api/Gateway.md

```diff
@@ -0,0 +1,163 @@
+Gateway API
+===========
+
+This document contains detailed descriptions of the gateway's API routes. For
+an overview of the gateway's API routes, see
+[API.md#gateway](/doc/API.md#gateway).  For an overview of all API routes, see
+[API.md](/doc/API.md)
+
+There may be functional API calls which are not documented. These are not
+guaranteed to be supported beyond the current release, and should not be used
+in production.
+
+Overview
+--------
+
+The gateway maintains a peer to peer connection to the network and provides a
+method for calling RPCs on connected peers. The gateway's API endpoints expose
+methods for viewing the connected peers, manually connecting to peers, and
+manually disconnecting from peers. The gateway may connect or disconnect from
+peers on its own.
+
+Index
+-----
+
+| Route | HTTP verb |
+| ----- | --------- |
+| [/gateway](#gateway-get) | GET |
+| [/gateway/connect/{netaddress}](#gatewayconnectnetaddress-post) | POST |
+| [/gateway/disconnect/{netaddress}](#gatewaydisconnectnetaddress-post) | POST |
+
+Examples
+--------
+
+* [gateway info](#gateway-info-example)
+* [connecting to a peer](#connect-example)
+* [disconnecting from a peer](#disconnect-example)
```

Comment:
I think there's an advantage of having the example link next to each route description (or in the title), as a developer is more likely to want to look at an example when they are looking through the request or response specs.